NDB.Covid19/NDB.Covid19.Droid/Resources/layout/welcome_page_five.xml

```diff
@@ -22,6 +22,8 @@
             android:id="@+id/consent_info_layout"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:paddingStart="24dp"
```

Comment:
use values from dimens (defaultHotizontalMargins) and it will be 16, not 24

Reply:
done